docs-v2/guides/api-authorization/re-authorize-a-connection.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: 'Re-authorize a connection'
+sidebarTitle: 'Re-authorize a connection'
+description: 'Guide on how to re-authorize a connection.'
+---
+
+There are cases where you need to re-authorize a connection, e.g. credentials have expired or the scopes/permissions have changed.
+
+The simplest way is to delete and recreate the connection, but this will reset the connection's attached data ([metadata](/guides/customer-configuration#store-customer-specific-data), [synced records](/guides/syncs/overview#sync-endpoints-and-data-caching)). Most of the time when testing, this is fine. 
+
+Follow the instructions below if you need to re-authorize a connection without altering it's attached data.
+
+# Re-authorize a connection from your app
+
+<Tip>
+This is the recommended way to re-authorize **production** connections, so customers can re-authorize indenpently, without having to manually send you their account credentials.
+</Tip>
+
+You can implement the re-authorization flow in your app by having your backend call a different endpoint when the connection already exists.
+
+Instead of generating the [default endpoint](/guides/api-authorization/authorize-in-your-app-default-ui#1-generate-a-session-token-backend) to generate a session token, use the `POST /connect/sessions/reconnect` endpoint ([API](/reference/api/connect/sessions/reconnect)/[SDK](/reference/sdks/node#create-a-reconnect-session) reference) to generate a rsession token meant for reconnection. This token is then used on the frontend in the same way as when creating a connection, but it will reconnect the existing connection instead.
```

Comment:
```suggestion
Instead of generating the [default endpoint](/guides/api-authorization/authorize-in-your-app-default-ui#1-generate-a-session-token-backend) to generate a session token, use the `POST /connect/sessions/reconnect` endpoint ([API](/reference/api/connect/sessions/reconnect)/[SDK](/reference/sdks/node#create-a-reconnect-session) reference) to generate a session token meant for reconnection. This token is then used on the frontend in the same way as when creating a connection, but it will reconnect the existing connection instead.
```